assets/css/src/components/_datepicker.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../../../node_modules/react-datepicker/src/stylesheets/datepicker';
+
+.mailpoet-datepicker {
+  display: inline-block;
+  vertical-align: middle;
+
+  .react-datepicker {
```

Comment:
Add `left: -1px;` so it's aligned perfectly with an input (otherwise it's positioned from the border resulting in 1px shift).